### DISCUSSION TITLE
Starter Assignment

### DISCUSSION THREAD

**Discussion by jrglass6** [community/community#44397] [GitHub Education]
I'm trying to import the starter assignment into my GitHub Classroom but there is an issue with the import I can't resolve. Has anyone else had this problem when they set up their first Classroom? Please help!
<img width="425" alt="Screen Shot 2023-01-14 at 11 13 22 AM" src="https://user-images.githubusercontent.com/22840060/212557067-211425c0-2bba-4948-84ea-4969101d8c1f.png">

**Comment by nwhysel** [comment]
I’m having a similar problem. Except the import works, and I can give it a name and due date. It just doesn’t show up in my classroom. It gives an error: “Unable to create assignment at this time, please try again”
